Source/Core/Core/Boot/Boot.cpp

```diff
@@ -226,7 +226,11 @@ bool CBoot::BootUp()
 	NOTICE_LOG(BOOT, "Booting %s", _StartupPara.m_strFilename.c_str());
 
 	g_symbolDB.Clear();
-	VideoInterface::Preset(_StartupPara.bNTSC);
+
+	// PAL Wii uses NTSC framerate and linecount in 60Hz/480i and Progressive/480p modes
```

Comment:
`Progressive/480p` should be `60Hz/480p`. Progressive is already denoted by the p, and the framerate is useful to know.